main.go

```diff
@@ -0,0 +1,127 @@
+package authenticated_block_store
+
+import (
+	"fmt"
+	"log"
+	"io/ioutil"
+        "strings"
+        "os"
+        "crypto/sha256"
+        "math/rand"
+        "time"
+
+)
+
+//size of each block, intended to be 4KB TODO
```

Comment:
Does it work to submit this code with 4k here? If so, just do it. If not, this should explain why.